js2-mode.el

```diff
@@ -7264,7 +7264,7 @@ are ignored."
            when (and (eq 'block (js2-comment-node-format node))
                      (save-excursion
                        (goto-char (js2-node-abs-pos node))
-                       (looking-at "/\\*global ")))
+                       (looking-at "/\\* ?+global ")))
```

Comment:
I was using regexp builder in Emacs and I was nearly certain that it matched things it shouldn't. 

I'll double check.